modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+hosting cluster:: See _management cluster_.
+
+hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted. In some contexts, the management cluster is referred to as the _hosting cluster_.
+
+management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+[id="hosted-control-planes-personas_{context}"]
+== Personas
```

Comment:
One question I have about these personas are if they are just personas/typical ways certain users use hosted control planes) or if they are actually official roles that are set somewhere. The way is reads now to me is the former, but then it talks about legitimate RBAC permissions, etc. that makes me think they might be real roles (and if they are, then it should probably list its actual official role, like how we do `cluster-admin`, etc.).

Reply:
Not real roles. The real roles are the things that the personas can assume, for example, cluster-admin. This is more "thought leadership" and alignment to allow us to use common language to discuss use-case. Also what we are pushing upstream.